File: src/domain/core/item.ts
export class Item {
  constructor(
    public id: number,
    public sellerId: number,
    public name: string,
    public description: string,
    public price: number,
    public imageUrl: string,
    public isSold: boolean,
    public delete_time: Date | null,
  ) {}
}

File: WebApplication/Domain/Entities/Item.cs
using WebApplication.Domain.ValueObject;

namespace WebApplication.Domain.Entities;

public class Item
{
    public ItemId Id { get; set; }
    public UserId SellerId { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal Price { get; set; }
    public string ImageUrl { get; set; }
    public bool IsSold { get; set; }
    public DateTime DeleteTime { get; set; }
}